src/main.rs

```diff
@@ -15,6 +20,32 @@ fn parse_matches() -> ArgMatches<'static> {
                 .default_value("default")
                 .help("namespace to deploy test cluster"),
         )
+        .arg(
+            Arg::with_name("deploy_method")
+                .long("deploy-method")
+                .takes_value(true)
+                .possible_values(&["local", "tar", "skip"])
+                .default_value("local")
+                .help("Deploy method. tar, local, skip. [default: local]"),
```

Comment:
nit: You shouldn't need to specify the default in the help text, I think clap does that for you